src/handlePreviewToggles.js

```diff
@@ -19,20 +19,8 @@ module.exports = function(args) {
       console.log("Enabling preview feature", clc.bold(args[1]) + "...");
       previews[args[1]] = true;
       configstore.set("previews", previews);
-      var tokens = configstore.get("tokens");
-
-      var next;
-      if (tokens && tokens.refresh_token) {
-        next = auth.logout(tokens.refresh_token);
-      } else {
-        next = Promise.resolve();
-      }
-      return next.then(function() {
-        console.log("Preview feature enabled!");
-        console.log();
-        console.log("Please run", clc.bold("firebase login"), "to re-authorize the CLI.");
-        return process.exit(0);
-      });
+      console.log("Preview feature enabled!");
+      return process.exit(0);
```

Comment:
Odd question, but does this _have_ to do `process.exit`? It looks like the return value of this is used by `bin/firebase.js` so returning `true` should give the same behavior?

Reply:
Your guess is as good as mine, but usually these kinds of things are load-bearing kludges.

![I have no memory of this place](https://media1.tenor.com/images/4b5badb33a987d38e6d56f6796c1988c/tenor.gif?itemid=5385535)